curriculum/challenges/english/blocks/workshop-major-browsers-list/68ecfa1003d88990bb96a33d.md

```diff
@@ -0,0 +1,85 @@
+---
+id: 68ecfa1003d88990bb96a33d
+title: Step 3
+challengeType: 0
+dashedName: step-3
+---
+
+# --description--
+Add your first term and description.
+
+Inside your `<dl>`, add a `<dt>` element for **Google Chrome**, followed by a `<dd>` element dscribing it.
+
+> "This is a free web browser developed by Google and first released in 2008."
```

Comment:
```suggestion
`This is a free web browser developed by Google and first released in 2008.`
```

put text that needs to be added as is in backticks, please apply this to all the steps